packages/flutter_tools/bin/fuchsia_tester.dart

```diff
@@ -162,7 +160,7 @@ Future<void> run(List<String> args) async {
       // package (i.e. contains lib/ and test/ sub-dirs). In some cases,
       // test files may appear to be in the root directory.
       if (coverageDirectory == null) {
-        globals.fs.currentDirectory = testDirectory.parent;
+        globals.fs.currentDirectory = testDirectory!.parent;
       } else {
         globals.fs.currentDirectory = testDirectory;
```

Comment:
```suggestion
        globals.fs.currentDirectory = testDirectory!;
```

Reply:
I tried that, but the analyzer complains that this is an unnecessary assertion.

Reply:
I see, I guess because `set currentDirectory` takes a dynamic. This is fine.